keps/sig-node/24-apparmor-ga/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: AppArmor graduation to General Availability (GA)
+kep-number: 24
+authors:
+  - "@saschagrunert"
+  - "@tallclair"
+owning-sig: sig-node
+participating-sigs:
+  - sig-api-machinery
+  - sig-auth
+status: implementable
```

Comment:
shouldn't be "implementable" with "TBD" approvers and no PRR review :-)

Reply:
Fixed :+1: